example/lib/main.dart

```diff
@@ -170,10 +170,13 @@ class _MyHomePageState extends State<MyHomePage> {
       maxSelectedDate: _currentDate.add(Duration(days: 360)),
       todayButtonColor: Colors.transparent,
       todayBorderColor: Colors.green,
+      daysHaveCircularBorder: false,
```

Comment:
I am not sure why this is needed!

Reply:
Looks like we can remove it

Reply:
why this change has not added it's more power full to customize the design
![image](https://user-images.githubusercontent.com/16919117/106896047-b7b64e00-66f9-11eb-9289-e855b3e9bce7.png)

Reply:
@eslamwael74 Because `daysHaveCircularBorder` doesn't do anything here.